packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -463,7 +463,7 @@ export interface IRootMergeBlock extends IMergeBlock {
  * @internal
  */
 export class MergeTree {
-    private static readonly zamboniSegmentsMaxCount = 2;
+    static readonly zamboniSegmentsMaxCount = 2;
```

Comment:
move this to a constant exported from the zamboni file `export const zamboniSegmentsMaxCount = 2`

Reply:
i think this can just be deleted as you have the const in zamboni.ts now